src/tests/modify.test.js

```diff
@@ -586,3 +649,118 @@ describe('modify() - create fields', () => {
     ]);
   });
 });
+
+describe('modify() - pick fields', () => {
+  it('basic usage', () => {
+    const { schema, warnings } = modify(schemaTickets, {
```

Comment:
I've rebased this (Pick PR) with main https://github.com/remoteoss/json-schema-form/pull/83, including the new specs discussed in the previous threads. Ready for a last review :)